tasks.py

```diff
@@ -57,6 +57,22 @@ def server(host=None, port=5000, debug=True, live=False):
 
 
 @task
+def debug_server(host=None, port=5000, debug=True, live=False):
```

Comment:
That's definitely legacy stuff I need to get rid of. I wrote that code before I figured out how to use import pdb; ipdb.set_trace().